config/karma/karma.tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "declaration": false,
+    "noImplicitAny": false,
+    "sourceMap": false,
+    "inlineSourceMap": true,
+    "jsx": "react",
+    "noLib": false,
+    "suppressImplicitAnyIndexErrors": true,
+    "types": [
+      "karma-chai-sinon",
+      "webpack-env"
+    ],
+    "typeRoots": [
+      "../../node_modules/@types"
```

Comment:
Because `karma.tsconfig.json` is inside `config/karma/`, there is not `node_modules` folder in the same level, so `tsc` complains about typings:

```
ERROR in [at-loader] TS2688: Cannot find type definition file for 'karma-chai-sinon'.

ERROR in [at-loader] TS2688: Cannot find type definition file for 'webpack-env'.

ERROR in [at-loader] ./src/app.tsx:6:21
    TS2304: Cannot find name 'require'.
```